infra/src/main/kotlin/fi/fta/geoviite/infra/authorization/UserData.kt

```diff
@@ -24,49 +24,31 @@ data class Role(val code: Code, val name: AuthName, val privileges: List<Privile
 
 data class Privilege(val code: Code, val name: AuthName, val description: FreeText) : GrantedAuthority {
     @JsonIgnore
-    override fun getAuthority(): String = code.value
+    override fun getAuthority(): String = code.toString()
```

Comment:
Vaihdoin sisäisen value-muuttujan privaatiksi koska näitä on tarkoitus käyttää yhteneväisesesti stringien kanssa. toString:llä saa joka tapauksessa ihan String-olion silloin kun sillä on väliä.